src/scss/tooltip.scss

```diff
@@ -17,7 +17,7 @@
 
```

Comment:
These changes fix the close button moving to the bottom when the tooltip content has multiple lines of text.